dev/com.ibm.ws.messaging.runtime/src/com/ibm/ws/sib/admin/internal/JsLocalizer.java

```diff
@@ -269,18 +269,18 @@ private MasterEntry updateLpMaps(LWMConfig lp) {
         LocalizationEntry lEntry = new LocalizationEntry(ld);
         lpMap.put(lpName, lEntry);
         String destName = lpName.substring(0, lpName.indexOf("@"));
-        MasterEntry masterEntry = masterMap.get(destName);
+        MainEntry MainEntry = mainMap.get(destName);
```

Comment:
variable name should be mainEntry instead of MainEntry

Reply:
fixed!